addons/binding/org.openhab.binding.meterreader/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="meterreader" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="meter">
+		<label>Meter device</label>
+		<description>The meter device to read the SML or IEC 62056-21 messages from</description>
+
+		<config-description>
+			<parameter name="port" type="text">
+				<label>Connected serial port</label>
+				<description>The device serial port (e.g. /dev/tty0 or COM1)</description>
+				<required>true</required>
```

Comment:
you can add
```
<context>serial-port</context>
```
which will make the list of available ports being shown in the Paper UI.

Reply:
I've implemented this functionality manually before this feature was available in openhab.
But for sure I will rather go for the integrated functionality hence will change it!

Reply:
Done!